sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Filtering/Filter.cs

```diff
@@ -19,7 +19,7 @@ namespace Azure.Monitor.OpenTelemetry.LiveMetrics.Internals.Filtering
     /// The filter's configuration (condition) is specified in a <see cref="FilterInfo"/> DTO.
     /// </summary>
     /// <typeparam name="TTelemetry">Type of telemetry documents.</typeparam>
-    internal class Filter<TTelemetry>
+    internal class Filter<TTelemetry> where TTelemetry : DocumentIngress
```

Comment:
Note that this is the file with all the AOT warnings.
Once I added this `where` constrain, all the other generic classes that didn't have the same constraint threw errors.
That's why you'll see several other classes where the only change is the where constraint.